server/models/like.model.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from 'mongoose';
+import httpStatus from 'http-status';
+import APIError from '../helpers/APIError';
+import config from '../../config/config';
+
+/**
+ * @swagger
+ * definitions:
+ *  Like:
+ *    type: object
+ *    properties:
+ *      _id:
+ *        $ref '#/definitions/ObjectId'
+ *      __v:
+ *        $ref: '#/definitions/MongoVersion'
+ *      userId:
+ *        $ref '#/definitions/ObjectId'
+ *      postId:
+ *        $ref '#/definitions/ObjectId'
+ *      dateCreated:
+ *        type: string
+ *        format: date-time
+ *        description: Date comment created
+ *        example: 2017-10-30T01:05:39.674Z
+ */
+const LikeSchema = new mongoose.Schema({
+  userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+  postId: { type: mongoose.Schema.Types.ObjectId, ref: 'Post' },
+});
```

Comment:
Missing `createdAt` ? 
Other entities have a field named dateCreated. It seems to be the most repeated. I found a createdAt in users. In fact, I don't know if we can conclude that there is a pattern. It's up to you

Reply:
Yea, I'm use to `createdAt` as well, but I've been seeing `dateCreated` throughout the codebase. I was trying to match how votes are handled and that model didn't have a creation property so I didn't add it initially. I'll go ahead and add it.